### PR TITLE
Mozfest call for proposals

### PR DESCRIPTION
**Pull request by TheoChevalier**

### CODE REVIEW

File: en-US/emails/2017/mozfest_call.lang [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+;Your session is documented in this issue:
+Your session is documented in this issue:
+
+
+;We encourage you to follow the curation process. You can chat with Space Wranglers (the people who curate sessions for each MozFest space), ask questions and develop your session with a network of collaborators in the comments section of your issue.
+We encourage you to follow the curation process. You can chat with Space Wranglers (the people who curate sessions for each MozFest space), ask questions and develop your session with a network of collaborators in the comments section of your issue.
+
+
+;If you’re new to GitHub, this 45 second video will walk you through signing up for a free account.
+If you’re new to GitHub, this 45 second video will walk you through signing up for a free account.
```
Comment (flodolo) [inline en-US/emails/2017/mozfest_call.lang]: typo: seconds

File: en-US/other/2017/mozfest_form.lang [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+;Twitter handle
+Twitter handle
+
+
+;GitHub handle
+GitHub handle
+
+
+;Additional facilitators for your session (separate multiple name with a comma: Lois Haibt, Feng-hsiung Hsu, Alan Turing)
+Additional facilitators for your session (separate multiple name with a comma: Lois Haibt, Feng-hsiung Hsu, Alan Turing)
```
Comment (flodolo) [inline en-US/other/2017/mozfest_form.lang]: typo: name -> names

File: en-US/other/2017/mozfest_form.lang [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+;Festival Spaces
+Festival Spaces
+
+
+;At MozFest, sessions are thematically organised into spaces. Please read the brief descriptions of each space <a>here</a> before answering the following questions.
+At MozFest, sessions are thematically organised into spaces. Please read the brief descriptions of each space <a>here</a> before answering the following questions.
+
+
+;What space do feel your session will best contribute to?
+What space do feel your session will best contribute to?
```
Comment (flodolo) [inline en-US/other/2017/mozfest_form.lang]: Uhm, English? Maybe missing a 'you' (do you feel).

File: en-US/social/2017/mozfest.lang [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+## MAX_LENGTH: 140
+# LIMIT: 140 characters (will be used as a tweet). Feel free to rephrase as needed to fit.
+;A healthy Internet belongs to all of us. Help protect online freedom and opportunity at #MozFest: mzl.la/mozfest
+A healthy Internet belongs to all of us. Help protect online freedom and opportunity at #MozFest: mzl.la/mozfest
+
+
+## MAX_LENGTH: 140
+# LIMIT: 140 characters (will be used as a tweet). Feel free to rephrase as needed to fit.
+;Join the open internet movement and submit session proposal for #MozFest: mzl.la/mozfest
+Join the open internet movement and submit session proposal for #MozFest: mzl.la/mozfest
```
Comment (flodolo) [inline en-US/social/2017/mozfest.lang]: I feel like there should be an "a" before session proposal

File: en-US/other/2017/mozfest_form.lang (hunk 3 above)
Comment (TheoChevalier) [inline en-US/other/2017/mozfest_form.lang]: Already asked, I’ve been told it’s correct

Comment (flodolo) [inline en-US/other/2017/mozfest_form.lang]: 🐮 (meh)

**Comment by TheoChevalier** [comment]
@koehlermichael Just sent you an email, but additional heads up: you can ignore those strings

**Pull request merged**